.travis.yml

```diff
@@ -2,9 +2,6 @@ language: node_js
 node_js:
   - "10"
   - "8"
-  - "5"
```

Comment:
Do we have any apps on Node 8? Could we remove support for that as well? :thinking:

Reply:
I removed 4/5 based on a conversion with @kiyanwang, but I'm happy to remove 8 as well if we're not using it

Reply:
We do have apps/buils running node 8 so I wouldn't remove that just yet. (echo is a case in point)